rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] Kindly edit the accessor to show `rental` belongs to a `book` and also belongs to a `person`

### Expected:
``attr_accessor :date, :book, :person``